### ISSUE TITLE
Action link in a Care card (Immediate)

### ISSUE THREAD

**Issue by davidhunter08** [nhsuk/nhsuk-frontend#588]
When using an action link inside an immediate care card, the green action link icon does not have sufficient contrast with the black background.

<img width="658" alt="Screenshot 2020-02-25 at 12 00 13" src="https://user-images.githubusercontent.com/14331000/75245975-c47eb200-57c6-11ea-852a-4fe744d6ce50.png">

**Comment by andrewbarnesweb** [comment]
It's only just missing at 2.8 with a target of 3 🙁: https://snook.ca/technical/colour_contrast/colour.html#fg=007F3B,bg=212B32FFFFFF

The only other "green" I can see in the palette is: `$color_nhsuk-aqua-green`. Which gives you a 4.6: https://snook.ca/technical/colour_contrast/colour.html#fg=00A499,bg=212B32FFFFFF

![image](https://user-images.githubusercontent.com/20924669/75720704-e13b4e00-5cce-11ea-941c-2fae6045f7b9.png)

**Comment by AdamChrimes** [comment]
Few design options...

`$color_nhsuk-white`

<img width="680" alt="Screenshot 2020-03-19 at 14 33 41" src="https://user-images.githubusercontent.com/11615501/77078567-a9cae200-69ee-11ea-858a-039b60905f02.png">

`$color_nhsuk-red`

<img width="680" alt="Screenshot 2020-03-19 at 14 34 49" src="https://user-images.githubusercontent.com/11615501/77078675-d3840900-69ee-11ea-8651-e50dc813ed1c.png">

`$color_nhsuk-yellow`

<img width="673" alt="Screenshot 2020-03-19 at 14 36 35" src="https://user-images.githubusercontent.com/11615501/77078844-0cbc7900-69ef-11ea-8d22-a358c11898aa.png">

**Comment by mcheung-nhs** [comment]
These are what the various colours look like when in high contrast mode in Windows

![action-link-emergency-high-contrast-windows](https://user-images.githubusercontent.com/4749968/78166394-9daa4000-7444-11ea-9b5c-2f413a5a0374.png)

and in Chrome using the [high contrast plugin](https://chrome.google.com/webstore/detail/high-contrast/djcfdncoelnlbldjfhinnjlhdjlikmph?hl=en)

![action-link-emergency-high-contrast-chrome](https://user-images.githubusercontent.com/4749968/78166527-d8ac7380-7444-11ea-8e17-8db9d260d5a5.png)

**Comment by davidhunter08** [comment]
`$color_nhsuk-white` looks to be the best option.
